docs/global/authentication.md

```
@@ -661,3 +661,134 @@ All authentication events are logged with:
+    recaptchaToken: await this.#getRecaptchaToken()
+};
+```
+
+**Services**:
+- `PublicTenantConfigService`: Loads config from `/api/config/public`, caches in sessionStorage
+- `RecaptchaService`: Loads Google script, executes reCAPTCHA, returns token
+
+### Testing
+
+**Development** (Google test keys - always pass):
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)